docker/common/install_base.sh

```diff
@@ -112,7 +118,11 @@ install_python_rockylinux() {
 }
 
 install_pyp_rockylinux() {
-  bash -c "pip3 install 'urllib3<2.0' pytest"
+  EXTRA_INDEX_URL=""
+  if [ -n "${GITHUB_MIRROR}" ]; then
+    EXTRA_INDEX_URL="--extra-index-url https://urm.nvidia.com/artifactory/api/pypi/sw-tensorrt-pypi/simple"
+  fi
+  bash -c "pip3 install ${EXTRA_INDEX_URL} 'urllib3<2.0' pytest"
```

Comment:
```suggestion
  if [ -n "${GITHUB_MIRROR}" ]; then
      PIP_INDEX_URL ="https://urm.nvidia.com/artifactory/api/pypi/pypi-remote/simple"
  fi
  bash -c "pip3 install 'urllib3<2.0' pytest"
```

I think this approach might be better. It applies to all "pip install" commands. The new URL also doesn't contain internal packages.

We also need to make sure `PIP_INDEX_URL` isn't set in the generated NGC containers.